Source/WebKit/UIProcess/Extensions/WebExtensionMatchPattern.h

```diff
@@ -101,8 +101,10 @@ class WebExtensionMatchPattern : public API::ObjectImpl<API::Object::Type::WebEx
 
     unsigned hash() const { return m_hash; }
 
+#ifdef __OBJC__
     _WKWebExtensionMatchPattern *wrapper() const { return (_WKWebExtensionMatchPattern *)API::ObjectImpl<API::Object::Type::WebExtensionMatchPattern>::wrapper(); }
```

Comment:
Would be nice to use `dynamic_objc_cast<_WKWebExtensionMatchPattern>(...)` here from `<wtf/cocoa/TypeCastsCocoa.h>`.